tox.ini

```diff
@@ -1,17 +1,19 @@
 [tox]
 envlist =
-    py{36,37,38,39,310}-django-22
-    py{36,37,38,39,310}-django-32
-    py{38,39,310}-django-40
-    py{38,39,310}-django-main
+    py{36,37,38,39,310}-django22
+    py{36,37,38,39,310}-django32
+    py{38,39,310}-django40
+    py{38,39,310}-django41
+    py{38,39,310}-djangomain
 
 [testenv]
 passenv = *
 commands =
     coverage run manage.py test
 deps =
-    py{36,37,38,39,310}-django-22: Django>=2.2,<2.3
-    py{36,37,38,39,310}-django-32: Django>=3.2,<3.3
-    py{38,39,310}-django-40: Django>=4.0,<4.1
-    py{38,39,310}-django-main: https://github.com/django/django/archive/main.tar.gz
+    django22: Django>=2.2,<2.3
+    django32: Django>=3.2,<3.3
+    django40: Django>=4.0,<4.1
+    django41: Django>=4.1,<4.2
+    djangomain: https://github.com/django/django/archive/main.tar.gz
```

Comment:
+1 for adding Django 4.1 but why remove the dash characters? They make things just a tiny bit easier to read.

IMO the dashes should stay. I'll leave others to chime in on this as well.

Reply:
As I said in the PR title, tox sees names as tags separated by dashes. By putting a dash between `django` and the version number, you can't use `django<V>` as a tag, which I've used to simplify `deps` here. Putting the dash back in will require the previous format of repeating the full environment names, which seems error prone.

Reply:
Is it possible to use underscore instead? To keep the readability and allow to use that as a tag?

Reply:
I don't know, feel free to experiment and make a suggestion on github if you really care about it.